### PR TITLE
Add a short overview of the COOL to the README

### PR DESCRIPTION
**Pull request by hillaryj**
#  #

## 🗣 Description ##

Adds a blurb to the README and moves the wiki link upwards to provide a small intro to folks who may not know what the COOL is or does.

## 💭 Motivation and context ##

This is a suggested quality-of-life improvement so newbies don't have to know to navigate to the wiki to get an overview.

## 🧪 Testing ##

Documentation only; linters pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,18 @@
 
 Welcome to the CISA's Cloud Optimized Operations Lab (COOL).
 
+The COOL provides ephemeral environments for performing
+cybersecurity assessments. Each assessment is isolated from other
+assessments and is accessed via a browser-based remote desktop after
```
Comment (jsf9k) [inline README.md:13]:
```suggestion
cybersecurity assessments. The assessment environments are isolated
from each other and are accessed via a browser-based remote desktop after
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,18 @@
...
+authenticating into the COOL VPN via multi-factor authentication.
+After an assessment is complete, all data is archived for future use
+and the assessment environment is destroyed.
```
Comment (jsf9k) [inline README.md:16]:
```suggestion
After an assessment is complete, all relevant data is archived for 
future use and the assessment environment is destroyed.
```

**Review by jsf9k — APPROVED** [review]
Approved.  I left a couple of minor wording suggestions that you can take or leave.

**Review by dav3r — APPROVED** [review]
Nice changes!  📚 
I agree with @jsf9k's suggestions.

**Pull request merged**